spec/command/deintegrate_spec.rb

```diff
@@ -1,40 +1,52 @@
 require File.expand_path('../../spec_helper', __FILE__)
 
-def fixture_project(named)
```

Comment:
These changes are mostly just refactoring to clean up adding specs for more versions of CocoaPods.